render.h

```diff
@@ -47,6 +49,12 @@ void draw_message(int severity, char line1[30], char line2[30], char line3[30],
 //this will only indicate how much % are left. Mavlink specific, but could be used with others as well.
 void draw_batt_gauge(int remaining, float pos_x, float pos_y, float scale);
 void draw_batt_status(float voltage, float current, float pos_x, float pos_y, float scale);
+
+// totals
+void draw_TOTAL_AMPS(float current, float pos_x, float pos_y, float scale);
+void draw_TOTAL_DIST(int gpsspeed, float pos_x, float pos_y, float scale);
```

Comment:
@Yes21 here are the int gpsspeed that I think will need to be changed to the relevant int that you most likely will have to declare.

Reply:
Euh ... I think I don't understand.
Sorry !!

Reply:
😀 no worries. I am no expert either

“Int gpsspeed” should be something like “float total_distance” then in one of the other scripts where these int and float are defined you will have to declare your new float variable (or int if you decide to stay with that) . Unless total distance calculated somewhere else already? Then you need that variable

Reply:
Ok just looked in render.h

It appears to be “int distance”

Reply:
It just occurred to me- are you trying to calculate the total distance flown? Sorry if I am misunderstanding. You do not want just the distance to home. You are trying to calculate the sum of all distance you have flown on that flight?

Reply:
Ok I think I understand what you are trying to do. I apologize I just read the issue you opened and did not go all the way back to the original pr in ez by wowie.

It has the potential to be quite a large pr. I see that you have copied wowies render.h and osd config.txt files. But the real work is happening in render.c. I will spend some real time on this today and mostly tomorrow. Wowies pr for render.c is quite long and will also have to see how different it is with what Jelle actually did 😋

Reply:
All was already done in render.c !!
#1 (comment)

Reply:
So in render.c I do not see draw_total_amps, draw_total_dist, or draw_total_time. See-

https://github.com/wowi69/EZ-WifiBroadcast/blob/develop/wifibroadcast_osd/render.c

Line 809

Reply:
So do you think it only needs to replace _**draw_TOTAL_AMPS, draw_TOTAL_DIST and draw_TOTAL_TIME**_ by _**draw_total_amps, draw_total_dist, or draw_total_time**_ ?

Reply:
I wouldn’t change what you already have. You just need to add the missing methods from wowies pr.

Of course there may be other stuff missing that we are not aware of. 

I start another trip for work tomorrow so I will not be able to help very much on the osd stuff... 😔

Reply:
Ok. Thanks Luke.
I will read again your posts, and try to understand what I have to do ...

Reply:
@Yes21 I threw a quick PR together. Hope it helps! I would just leave this PR alone and try testing